metadata/latest/Microsoft.CognitiveServices.Speech.Extension.Embedded.TTS.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Name": "Microsoft.CognitiveServices.Speech.Extension.Embedded.TTS",
```

Comment:
This is a new package and will require a CSV entry -- https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/602/Package-Onboarding-step-by-step